### PR TITLE
Symbol type

### PR DESCRIPTION
**Pull request by armino-dev**
I think that's it for this year :).
Happy holidays everyone!

### CODE REVIEW

**Comment by lighthousand** [comment]
Happy holidays to you, too ! 🍾 🎉

File: 1-js/04-object-basics/03-symbol/article.md [hunk 1]
```diff
@@ -36,12 +36,12 @@ alert(id1 == id2); // false
...
-If you are familiar with Ruby or another language that also has some sort of "symbols" -- please don't be misguided. JavaScript symbols are different.
+Dacă sunteți familiari cu Ruby sau alt limbaj de programare ce are de asemenea orice fel de "simboluri" -- nu vă lăsați păcălit. Simbolurile JavaScript sunt diferite.
 
-````warn header="Symbols don't auto-convert to a string"
-Most values in JavaScript support implicit conversion to a string. For instance, we can `alert` almost any value, and it will work. Symbols are special. They don't auto-convert.
+````warn header="Simbolurile nu se auto-convertesc la un șir"
+Cele mai multe valori în JavaScript suportă conversie implicită la un șir. De exemplu, putem `alert`-a aproape orice valoare și va funcționa. Simbolurile sunt speciale. Ele nu se auto-convertesc.
 
-For instance, this `alert` will show an error:
+De exemplu, următoare instrucțiune `alert` va genera o eroare:
```
Comment (lighthousand) [inline 1-js/04-object-basics/03-symbol/article.md]: Typo : 'următoare' -> 'următoarea'

File: 1-js/04-object-basics/03-symbol/article.md [hunk 2]
```diff
@@ -176,118 +176,118 @@ let clone = Object.assign({}, user);
...
 
-Symbols have two main use cases:
+Simbolurile au două cazuri principale de folosire:
 
-1. "Hidden" object properties.
-    If we want to add a property into an object that "belongs" to another script or a library, we can create a symbol and use it as a property key. A symbolic property does not appear in `for..in`, so it won't be occasionally processed together with other properties. Also it won't be accessed directly, because another script does not have our symbol. So the property will be protected from occasional use or overwrite.
+1. Proprietăți "ascunse" ale unui obiect.
+    Dacă vrem să adăugăm o proprietate într-un obiect care "aparține" altui script sau librărie, putem crea un simbol și îl putem folosi ca și proprietate cheie. O proprietate simbolică nu apare în `for..in`, motiv pentru care, nu va fi procesată accidental împreună cu alte proprietăți. De asemenea, nu va fi accesată direct, deoarece un alt script nu va avea simbolul nostru. În concluzie, proprietatea va fi protejată împotriva folosirii sau rescrierii accidentale.
```
Comment (lighthousand) [inline 1-js/04-object-basics/03-symbol/article.md]: Grammar suggestion :  "altui script sau librărie" --> "altui script sau biblioteci"

File: 1-js/04-object-basics/03-symbol/article.md [hunk 3]
```diff
@@ -176,118 +176,118 @@ let clone = Object.assign({}, user);
...
 
-Technically, symbols are not 100% hidden. There is a built-in method [Object.getOwnPropertySymbols(obj)](mdn:js/Object/getOwnPropertySymbols) that allows us to get all symbols. Also there is a method named [Reflect.ownKeys(obj)](mdn:js/Reflect/ownKeys) that returns *all* keys of an object including symbolic ones. So they are not really hidden. But most libraries, built-in functions and syntax constructs don't use these methods.
+Tehnic, simbolurile nu sunt 100% ascunse. Există metode ale limbajului JavaScript [Object.getOwnPropertySymbols(obj)](mdn:js/Object/getOwnPropertySymbols) care ne permit sa accesăm toate simbolurile. Mai există o metodă numită [Reflect.ownKeys(obj)](mdn:js/Reflect/ownKeys) care returnează  *toate* cheile unui obiect incluzându-le și pe cele simbol. Deci ele nu sunt ascunse cu adevărat. Dar cele mai multe librării, funcții incluse în limbaj și construcții de sintaxă nu folosesc aceste metode.
```
Comment (lighthousand) [inline 1-js/04-object-basics/03-symbol/article.md]: Grammar suggestion : "librării" --> "biblioteci"

**Review by lighthousand — CHANGES_REQUESTED** [review]

File: 1-js/04-object-basics/03-symbol/article.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (armino-dev) [reply 1-js/04-object-basics/03-symbol/article.md]: This change was already made with commit 4eb75e5.

**Review by lighthousand — APPROVED** [review]
Good job!

**Pull request merged**